### DISCUSSION TITLE
General codebase improvement thread

### DISCUSSION THREAD

**Discussion by Boorj** [orval-labs/orval#145] [Ideas]
🍶 Errors should be clarified
![image](https://user-images.githubusercontent.com/2145313/119272877-7313db80-bc32-11eb-8e44-d9fed610f259.png)

🍶 fs-extra, /lib/fs-utils, outputFile, writeFile
https://github.com/anymaniax/orval/blob/master/docs/src/lib/fs-utils.js here's a file used in a single place. 
https://github.com/anymaniax/orval/blob/master/src/core/writers/schemas.ts#L55 

Looks like there could be more consistent way to save output files. It might be joined with file paths resolving. If you use outputFile, that could be used everywhere, and since that no need to create directories.

**Comment by anymaniax** [comment]
fixed with `5.4.1`
